package.json

```diff
@@ -45,7 +45,6 @@
     "jest": "jest",
     "jest:coverage": "jest --coverage",
     "pretest": "npm run lint",
-    "preinstall": "npm run clean:all",
```

Comment:
I was under the impression this was intentional, so that `npm install` would always remove old modules and generate fresh dependencies. I know a few people have run into issues with `npm install` and old modules that cause the command to fail or behave in an odd way.